src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -703,17 +703,20 @@ export default class FilingHistoryList extends Vue {
 
   /** Called by File a Correction button to correct the subject filing. */
   protected async correctThisFiling (item: HistoryItemIF): Promise<void> {
+    // see also TodoList.vue:doResumeFiling()
     switch (item?.name) {
       case FilingTypes.INCORPORATION_APPLICATION:
       case FilingTypes.CHANGE_OF_REGISTRATION:
-      case FilingTypes.CONVERSION:
       case FilingTypes.CORRECTION:
       case FilingTypes.REGISTRATION:
         // correction via Edit UI
         this.currentFiling = item
         this.fileCorrectionDialog = true
         break
 
+      case FilingTypes.ANNUAL_REPORT:
+      case FilingTypes.CHANGE_OF_ADDRESS:
+      case FilingTypes.CHANGE_OF_DIRECTORS:
```

Comment:
These are redundant but I added them to show that we explicitly can correct them.